### PR TITLE
[FEATURE] how to validate without checkpoint

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Deploying Great Expectations in a hosted environment without file system or CLI
+---
+
+If you follow the steps of the [Getting Started](../tutorials/getting-started/intro) tutorial, you create a standard deployment of Great Expectations. By default, this relies on two components:
+
+#. The Great Expectations :ref:`command-line interface (CLI)<command_line>` to initialize a Data Context, create Expectation Suites, add Datasources, etc.
```
Comment (cdkini) [inline docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: Any `:ref:` tags are from `.rst` syntax and should be hyperlinked in the new `.md` docs.

Also these `#` should be a numbered list per the old docs.

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The following guide gives an overview of creating an in-code Data Context including defaults to help you more quickly set one up for common configurations:
+
+- [How to instantiate a DataContext without a YML file](../guides/setup/configuring-data-contexts/how-to-instantiate-a-data-context-without-a-yml-file)
+
+The following guides will contain examples for each environment we have tested out:
+
+- [AWS EMR](./how-to-instantiate-a-data-context-on-an-emr-spark-cluster)
+- [Databricks](./how-to-instantiate-a-data-context-on-databricks-spark-cluster)
+- TODO: Add reference to "Google Cloud Composer"
```
Comment (cdkini) [inline docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: I think we should leave these two references but just put a note about pending port over. If we delete them we might forget about them?

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- TODO: Add reference to "Astronomer"
+
+
+Step 2: Create Expectation Suites and add Expectations
+-------------------------------------------------------
+
+If you want to create an Expectation Suite in your environment without using the CLI, you can simply follow this guide: [TODO: Add content and reference to "How to Create a New Expectation Suite Without the CLI"].
+
+In order to store your Expectation Suites so you can load them for validation at a later point, you will need to ensure that you have an Expectation Store configured: [TODO: Add content and reference to "Configuring Metadata Stores"].
```
Comment (cdkini) [inline docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md:39]: I have a few of these TODOs in my own PRs. Let's discuss a standardized approach for these.

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Step 4: Use Data Docs
+----------------------
+
+Finally, if you would like to build and view Data Docs in your environment, please follow the guides for configuring Data Docs: [Options for hosting Data Docs](../tutorials/getting-started/customize-your-deployment#options-for-hosting-data-docs).
+
+Additional notes
+----------------
+
+If you have successfully deployed Great Expectations in a hosted environment other than the ones listed above, we would love to hear from you. Please comment on the Discuss post below, or reach out to us on `Slack <https://greatexpectations.io/slack>`_.
```
Comment (cdkini) [inline docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: Slack should be hyperlinked

**Review by cdkini — CHANGES_REQUESTED** [review]
Left a few comments about links but overall it looks great! After those changes I'll happily approve and we can merge :)

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md (hunk 2 above)
Comment (alexsherstinsky) [reply docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: @cdkini not a bad idea -- in the next commit.

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: Must be a miss... checking.

File: docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md (hunk 4 above)
Comment (alexsherstinsky) [reply docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md]: Done.

**Review by cdkini — APPROVED** [review]
LGTM!

**Pull request merged**